src/index/resolve/resolveTypescript.ts

```diff
@@ -0,0 +1,65 @@
+import { dirname, relative, resolve as resolvePath, extname } from "path";
+import { existsSync } from "fs";
+import { parse } from "url";
+import { Config } from "../config";
+
+function normalizePath(p: string) {
+  // Is extended length or has non-ascii chars (respectively)
+  if (/^\\\\\?\\/.test(p) || /[^\u0000-\u0080]+/.test(p)) {
+    return p;
+  }
+  // Normalize to forward slash and remove repeating slashes
+  return p.replace(/[\\\/]+/g, "/");
+}
+
+function isRelative(s: string) {
+  return s[0] === ".";
+}
+
+function isUrl(s: string) {
+  return parse(s).protocol !== null;
+}
+
+export function resolveTypescript(
+  filePath: string,
+  importPath: string,
+  config: Pick<Config, "tsconfig" | "root">
+) {
+  if (isRelative(importPath)) {
+    // if it's relative path do not transform
+    return importPath;
+  }
+
+  const { tsconfig, root } = config;
+  const { paths = {}, baseUrl: baseUrl_ = ".", rootDir } = tsconfig;
```

Comment:
What's the goal of `baseUrl: baseUrl_ = "."`?

Reply:
oops, I was using it for something, not needed anymore